packages/calcite-components/src/components/handle/handle.scss

```diff
@@ -23,13 +35,17 @@
 :host(:not([disabled])) .handle {
   @apply cursor-move;
   &:hover {
-    @apply bg-foreground-2 text-color-1;
+    background-color: var(--calcite-handle-background-color);
```

Comment:
Rather than re-apply the prop, try reassigning the variable on the host

Reply:
background-color needs two separate vars for default, hover and selected. I think text needs the same 3.